src/abstractdataframe/reshape.jl

```diff
@@ -18,11 +18,11 @@ Stacks a DataFrame; convert from a wide to long format
 
 
 ```julia
-stack(df::AbstractDataFrame, measure_vars, id_vars)
-stack(df::AbstractDataFrame, measure_vars)
-stack(df::AbstractDataFrame)
-melt(df::AbstractDataFrame, id_vars, measure_vars)
-melt(df::AbstractDataFrame, id_vars)
+stack(df::AbstractDataFrame, measure_vars, id_vars; measure_name=:variable, value_name=:value)
```

Comment:
`melt` from data.table calls the first argument `variable_name`. I wonder which one is best. (AFAICT only `value_name` exists in reshape2.)